src/test/java/core/basesyntax/service/implementation/ParseServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParseService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParseServiceImplTest {
+    private final ParseService parseService = new ParseServiceImpl();
+
+    @Test
+    public void parse_NullList_NotOk() {
+        try {
+            parseService.parse(null);
+        } catch (Exception e) {
+            Assert.assertSame(NullPointerException.class, e.getClass());

```

Comment:
It's bad practise to expect NPE in test, lets add some null check in method and throw some Runtime exception(fix for all tests that expect NPE)